app/Http/Controllers/Api/ReportbackController.php

```diff
@@ -52,4 +54,15 @@ public function store(ReportbackRequest $request)
 
         return $this->item($reportback, $code);
     }
+
+    /**
+     * Update reportbackitem(s) that already exists.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @return \Illuminate\Http\Response
+     */
+    public function updateReportbackItems(Request $request)
+    {
+        $this->reportbackService->updateReportbackItems($request->all());
```

Comment:
Thank you so much for providing such detailed instructions and also providing context/explanation behind this! Totally makes sense to do this - I wasn't exactly sure what to return - if it should be a message "Updated" or the item and was going to come back to it but totally forgot to! You're right to return the object so there is proof that the reportback item has been updated.

I followed your instructions and for step three added: 
```
        $item = $this->reportbackService->updateReportbackItems($request->all());

        return $this->item($item, 201, $this->itemTransformer);
```

One error that I'm getting right now: 
```
ErrorException in ResourceAbstract.php line 145:
Argument 1 passed to League\Fractal\Resource\ResourceAbstract::setMeta() must be of the type array, object given, called in /home/vagrant/DoSomething.org/rogue/app/Http/Controllers/Api/ApiController.php on line 65 and defined
```